public/server/db/cta.json

```diff
@@ -0,0 +1,3 @@
+{
+  "definition": "Call To Action"
```

Comment:
Since abbreve is somewhat of an informal library, we won't be taking "formal" abbreviations like these. Hope you understand.

Reply:
I have made the necessary adjustments. Thanks for the feedback